samples/nrf5340/netboot/README.rst

```diff
@@ -70,11 +69,10 @@ Building and running
 
 This sample can be found under :file:`samples/nrf5340/netboot/` in the |NCS| folder structure.
 
-Follow the steps below to include the sample as a child image in a :ref:`multi-image <ug_multi_image>` build that contains a network core application:
+Complete the following steps to include the sample as an image in a sysbuild project that contains a network core application:
```

Comment:
Since there is only one step now, can it be rephrased like this:
To include the sample as an image in a sysbuild project that contains a network core application, add MCUboot to the build by enabling the \`\`SB_CONFIG_BOOTLOADER_MCUBOOT\`\`, \`\`SB_CONFIG_SECURE_BOOT_NETCORE\`\`, and \`\`SB_CONFIG_NETCORE_APP_UPDATE\`\` sysbuild Kconfig options in the project. The build system includes the sample in the build automatically.

Reply:
Have reworded slightly